css/index.css

```diff
@@ -1,3 +1,559 @@
+    margin-bottom: 20px;
+    font-size: 30px;
+}
+
+
+.Digital-Design {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    border: 1px solid black;
+    background-color: #f4f3f4;
+    margin-right: 20px;
+}
+
+.UX { 
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    border: 1px solid black;
+    background-color: #f4f3f4;
+    margin-right: 20px;
+}
+
+
+
+/*.nav {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 20px;
+    text-decoration: none;
+}*/
+
+
+
+.cont {
+    width: 900px;
+    margin: 0 auto;
+}
+
+.header-image {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    margin: 0 auto;
+}
+
+.row1 {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between; 
+    margin-bottom: 20px;
+    margin-top: 20px;
+
+}
+
+.row2 {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between; 
+    margin-bottom: 20px;
+
+}
+
+.row3 {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between; 
+    margin-bottom: 20px;
+    
+}
+
+
+
+.Digital-Marketing {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    border: 1px solid black;
+    background-color: #f4f3f4;
+    margin-right: 20px;
+
+}
+
+.WebDev { 
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    border: 1px solid black;
+    background-color: #f4f3f4;
+}
+
+.IOSDev {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    border: 1px solid black;
+    background-color: #f4f3f4;
+    margin-right: 20px;
+
+}
+
+.Android {
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    justify-content: space-around;
+    border: 1px solid black;
+    background-color: #f4f3f4;
+}
+
+.Some-facts {
+    display: flex;
+    flex-direction: column;
+    align-content: space-between;
+    padding-right: 40px;
+    
+}
+
+.bottom-section {
+    display: flex;
+    flex-direction: row;
+    padding-top: 20px;
+}
+
+.para {
+    padding-bottom: 20px;
+}
+
+ul {
+    list-style-position: inside;
+    line-height: 1.5;
+}
+
+.copy {
+    display: flex;
+    justify-content: center;
+    padding-top: 65px;
+}
+
+
+
+button {
+    max-width: 0 150px;
+	background-color: white;
+    border: 1.5px solid black;
+    padding: 5px 35px;
+    text-align: center;
+    text-decoration: none;
+    font-size: 16px;
+    margin: 10px 0 10px 0;
+}
+
+/*media query Table Mode*/
+
+@media (max-width: 800px) {
+
+    .container {
+        width: 100%;
+    }
+
+    header {
+        display: flex;
+        flex-direction: column;
+        align-content: center;
+        margin: 0 auto;
+    }
+
+    h1 {
+
+        font-size: 80px;
+    }
+
+    .navigation {
+        display: flex;
+        flex-direction: column-reverse;
+        justify-content: center;
+        margin: 0 auto;
+    }
+
+    .logo {
+        display: flex;
+        justify-content: center;
```

Comment:
Flex properties only work on objects with other objects inside of them. When you address your cross axis problem you won't have to worry about the width. The default attribute for align items is stretch, meaning all items will take on the entire length of the cross axis. That's why your image is blowing up.